### DISCUSSION TITLE
I want to join the group, but the group is full

### DISCUSSION THREAD

**Discussion by ingale726** [milvus-io/milvus#22186] [Q&A and General discussion]
My wx is 809969882. Could you invite me into the group?

**Comment by yhmo** [comment]
We will contact you via wechat.

**Comment by echo-initial** [comment]
Where can I join WeChat group

**Reply by yhmo** [reply]
> Where can I join WeChat group

这样，你在微信上关注 Zilliz 公众号，然后在公众号里随便打开一篇近期文章，在每篇文章的底部都有一个二维码(用户交流群的二维码)，扫码进入。
